apps/dsb-client-gateway-api/src/app/modules/message/service/message.service.ts

```diff
@@ -352,7 +352,7 @@ export class MessageService {
       await this.ddhubMessageService.messagesSearch(
         topicsIds,
         channel.conditions.qualifiedDids,
-        clientId,
+        clientId + "." + fqcn,
```

Comment:
Is it better to use?
`${clientId}.${fqcn}`

Reply:
updated